### DISCUSSION TITLE
Condition based react-query

### DISCUSSION THREAD

**Discussion by Miteshdv** [TanStack/query#396] [General] [answered]
How we can achieve call a asynchronous function only on button click or on some prop value

**Comment by denisborovikov** [comment, marked as answer]
Do you mean [manual querying](https://github.com/tannerlinsley/react-query#manual-querying) and [dependent queries]( https://github.com/tannerlinsley/react-query#dependent-queries)?

**Reply by Miteshdv** [reply]
Yes , Manual Querying
